walrus/api.py

```diff
@@ -9,20 +8,52 @@ def hello():
     return "Hello World"
 
 
-def get_capital(country_name, raise_errors=False):
-    # Load the data
-    data = pd.read_csv(file_path)
+def find_capital(country_name, raise_errors=False):
+    # Dictionary to store country and capital as key-value pair
+    country_capital = {}
 
-    # Find the row where the 'CountryName' column matches the input country_name
-    country_data = data[data['CountryName'] == country_name]
+    # Open the csv file with the reading mode
+    with open(file_path, 'r') as file:
```

Comment:
We're opening up the file on every invocation of the function. This is usually never done for real projects. Is there a better way?